charm/ksql/templates/ksql-server.properties

```diff
@@ -0,0 +1,15 @@
+bootstrap.servers={{ bootstrap_servers }}
+listeners=http://{{ listener_addr }}
+state.dir=/var/snap/ksql-server/common/kafka-streams
+
+ssl.keystore.location={{ ca_keystore }}
+ssl.keystore.password={{ keystore_password }}
+ssl.truststore.location={{ client_keystore }}
+ssl.truststore.password={{ keystore_password }}
+ssl.key.password={{ keystore_password }}
+ssl.endpoint.identification.algorithm=HTTPS
+security.protocol=SSL
```

Comment:
Are these ssl.* and security.protocol settings ignored if kafka is unsecured plaintext? Or do we need a conditional?

Reply:
Added config